src/ixdat/techniques/ms.py

```diff
@@ -1,7 +1,57 @@
 """Module for representation and analysis of MS measurements"""
 
 from ..measurements import Measurement
+import numpy as np
 
 
 class MSMeasurement(Measurement):
     """Class implementing raw MS functionality"""
+
+    def __init__(self, name, **kwargs):
+        """Initializes a MS Measurement
+
+        Args:
+            name (str): The name of the measurement
+            calibration (dict): calibration constants whereby the key
```

Comment:
Can you put a TODO here reminding any reader that this is not the final calibration implementation?
Even for this stop-gap measure, I would kind of like calibration to also bridge from mass names "M2" to molecule names "H2". I guess it would have to be in the form `{"H2": {"M2": 1.95}}` or similar, or the information that the M2 signal is for H2 needs to be stored somewhere else.
I'm happy to add a bit so that it's what I think is the minimum acceptable, without re-writing what I hope a certain company will let us use.